src/ncp/ncp_spinel.cpp

```diff
@@ -473,12 +481,97 @@ otbrError NcpSpinel::HandleResponseForPropSet(spinel_tid_t      aTid,
     return error;
 }
 
+otbrError NcpSpinel::HandleResponseForPropInsert(spinel_tid_t      aTid,
+                                                 spinel_command_t  aCmd,
+                                                 spinel_prop_key_t aKey,
+                                                 const uint8_t    *aData,
+                                                 uint16_t          aLength)
+{
+    otbrError error = OTBR_ERROR_NONE;
+
+    switch (mWaitingKeyTable[aTid])
+    {
+    case SPINEL_PROP_IPV6_MULTICAST_ADDRESS_TABLE:
+        if (aCmd == SPINEL_CMD_PROP_VALUE_IS)
+        {
+            spinel_status_t status = SPINEL_STATUS_OK;
+
+            VerifyOrExit(aKey == SPINEL_PROP_LAST_STATUS, error = OTBR_ERROR_INVALID_STATE);
+            SuccessOrExit(error = SpinelDataUnpack(aData, aLength, SPINEL_DATATYPE_UINT_PACKED_S, &status));
+            otbrLogWarning("Failed to subscribe to multicast address on NCP, error:%s", spinel_status_to_cstr(status));
+        }
+        else if (aCmd == SPINEL_CMD_PROP_VALUE_INSERTED)
+        {
+            otbrLogInfo("Subscribed to multicast address on NCP successfully.");
+        }
+        else
+        {
+            ExitNow(error = OTBR_ERROR_INVALID_STATE);
+        }
+        break;
+    default:
+        break;
+    }
+
+exit:
+    return error;
+}
+
+otbrError NcpSpinel::HandleResponseForPropRemove(spinel_tid_t      aTid,
+                                                 spinel_command_t  aCmd,
+                                                 spinel_prop_key_t aKey,
+                                                 const uint8_t    *aData,
+                                                 uint16_t          aLength)
+{
+    otbrError error = OTBR_ERROR_NONE;
+
+    switch (mWaitingKeyTable[aTid])
+    {
+    case SPINEL_PROP_IPV6_MULTICAST_ADDRESS_TABLE:
+        if (aCmd == SPINEL_CMD_PROP_VALUE_IS)
+        {
+            spinel_status_t status = SPINEL_STATUS_OK;
+
+            VerifyOrExit(aKey == SPINEL_PROP_LAST_STATUS, error = OTBR_ERROR_INVALID_STATE);
+            SuccessOrExit(error = SpinelDataUnpack(aData, aLength, SPINEL_DATATYPE_UINT_PACKED_S, &status));
+            otbrLogWarning("Failed to unsubscribe to multicast address on NCP, error:%s",
```

Comment:
same as above, the warning is logged after success.

Reply:
Updated same as above!